src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1952,63 +1952,44 @@ because xbuild doesn't support framework reference assemblies.
   <!-- Create our intermediate directory for assemblies -->
   <MakeDir Directories="$(MonoAndroidIntermediateAssemblyDir)" Condition=" !Exists ('$(MonoAndroidIntermediateAssemblyDir)') " />
 </Target>
+
+<Target Name="_CollectPlatformJarFilenames">
+	<ItemGroup>
+		<_PlatformJars Include="mono.android.jar" />
+		<_PlatformJars Include="runtime.jar" />
+	</ItemGroup>
+</Target>
   
-<Target Name="_GetMonoPlatformJarPath">
-  <GetMonoPlatformJar TargetFrameworkDirectory="$(TargetFrameworkDirectory)">
-    <Output TaskParameter="MonoPlatformJarPath" PropertyName="MonoPlatformJarPath" />
-    <Output TaskParameter="MonoPlatformDexPath" PropertyName="MonoPlatformDexPath" />
-  </GetMonoPlatformJar>
+<Target Name="_GetMonoPlatformJarPath" DependsOnTargets="_CollectPlatformJarFilenames">
+  <GetMonoPlatformJars TargetFrameworkDirectory="$(TargetFrameworkDirectory)" JarFiles="@(_PlatformJars)">
+    <Output TaskParameter="MonoPlatformJarPaths" ItemName="MonoPlatformJarPaths" />
+    <Output TaskParameter="MonoPlatformDexPaths" ItemName="MonoPlatformDexPaths" />
+  </GetMonoPlatformJars>
 </Target>
 
 <Target Name="_AddStaticResources"
-		Inputs="$(MonoPlatformJarPath);$(_AndroidBuildPropertiesCache)"
+		Inputs="@(MonoPlatformJarPaths);$(_AndroidBuildPropertiesCache)"
 		Outputs="$(_AndroidStaticResourcesFlag)"
 		DependsOnTargets="$(_BeforeAddStaticResources);_GetMonoPlatformJarPath">
 	<CopyResource ResourceName="machine.config" OutputPath="$(MonoAndroidIntermediateAssetsDir)machine.config" />
   <CopyResource
     ResourceName="MonoRuntimeProvider.Bundled.java"
     OutputPath="$(MonoAndroidIntermediate)android\src\mono\MonoRuntimeProvider.java"
     Condition="'$(AndroidUseSharedRuntime)' != 'true'" />
-
-  <CopyResource
-    ResourceName="NotifyTimeZoneChanges.java"
-    OutputPath="$(MonoAndroidIntermediate)android\src\mono\android\app\NotifyTimeZoneChanges.java"
-    Condition="'$(AndroidIncludeDebugSymbols)' == 'True'" />
-
-  <CopyResource
-    ResourceName="Seppuku.java"
-    OutputPath="$(MonoAndroidIntermediate)android\src\mono\android\Seppuku.java"
-    Condition="'$(AndroidIncludeDebugSymbols)' == 'True'" />
```

Comment:
One problem with this change is here: *currently*, `Seppuku.java` is only included within the app when `$(AndroidIncludeDebugSymbols)`=True, generally Debug-configuration builds.

With this PR, `Seppuku.java` will *always* be present in *all* configuration builds.

Perhaps `%(_RuntimeOutput.RemoveItems)` could be used to partially exclude `Seppuku.java`?

Reply:
I can remove it from the `runtime.jar` and leave it in the `runtime_fastdev.jar` (which will be used for debug purposes)